package.json

```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/opensearch-project/opensearch-dashboards-functional-test",
   "dependencies": {
-    "@opensearch-dashboards-test/opensearch-dashboards-test-library": "^1.0.3"
+    "@opensearch-dashboards-test/opensearch-dashboards-test-library": "opensearch-project/opensearch-dashboards-test-library#main"
```

Comment:
I used the scoped name `@opensearch-dashboards-test` because of NPM publish. Does your solution work if we just call it opensearch-dashboards-test-library?

Reply:
we have import statement in code using ‘@opensearch-dashboards-test/opensearch-dashboards-test-library’, it throws error from there. To avoid changing src code, we can keep  the scoped name @opensearch-dashboards-test for now.